packages/data-explorer-ui/src/components/common/Alert/alert.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import * as React from "react";
+import { Alert } from "./alert";
+
+export default {
+  argTypes: {
+    children: {
+      control: "text",
+      description: "Inner text",
+    },
+    color: {
+      description: "Color of alert",
+    },
+    icon: {
+      control: "text",
+      description: "Top left icon. If provided will replace severity.",
+    },
+    severity: {
+      description: "Top left predefined icon. Combine with color.",
+    },
+    title: {
+      control: "text",
+      description: "Highlighted text",
+    },
+    variant: {
+      description: "[deprecated] Type of the alert",
```

Comment:
Hi @Ajalle - would you mind explaining the use of the word `[deprecated]` here in this context? 🤔